api/Makefile

```diff
@@ -18,3 +18,16 @@ export-document-plan:
 	     -d "{\"query\": \"{documentPlan(id:\\\"$(id)\\\",name:\\\"$(name)\\\",kind:\\\"Document\\\"){id uid name kind blocklyXml documentPlan}}\"}" | \
 	jq -r '.data.documentPlan' | \
 	jq '.documentPlan = (.documentPlan | fromjson)'
+
+export-all-document-plans:
+	@curl -s -X POST http://localhost:3001/_graphql \
+         -H "Content-Type: application/json" \
+         -d "{\"query\": \"{documentPlans{items{id uid name kind blocklyXml documentPlan}}}\"}" | \
```

Comment:
At least for me, this query only returns actual DocumentPlans, RGLs and AMRs are skipped